server/middleware.ts

```diff
@@ -0,0 +1,29 @@
+import * as express from "express";
+
+import config from "./config";
+
+export function isAdmin(
+	request: express.Request,
+	response: express.Response,
+	next: express.NextFunction
+) {
+	response.setHeader("Cache-Control", "private");
+	const auth = request.headers.authorization;
+
+	if (auth && typeof auth === "string" && auth.indexOf(" ") > -1) {
```

Comment:
Could use `includes` instead of `indexOf`, slightly clearer. What about the `basic` part of the header? Just curious if that matters.

Reply:
Doesn't really matter, but this is what registration did. I will change it to includes.